src/utils/useTimeout.js

```diff
@@ -0,0 +1,19 @@
+import { useRef, useEffect } from 'react';
+
+export default function useTimeout(callback, delay) {
```

Comment:
Could we maybe add a little bit of doc on what this custom hook(?) is for and where it is used? I am still not great at reading hook syntax so I'm not exactly sure what its purpose is?

Reply:
docs added!

Reply:
Thanks!